server/tests/ballot_comparison/snapshots/snap_test_ballot_comparison.py

```diff
@@ -100,7 +100,7 @@
 Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
 1,Contest 1,Targeted,22,No,0.1128631733,DATETIME,DATETIME,Choice 1-1: 5; Choice 1-2: 1\r
 1,Contest 2,Opportunistic,,No,0.1888821229,DATETIME,DATETIME,Choice 2-1: 4; Choice 2-2: 1; Choice 2-3: 0\r
-2,Contest 1,Targeted,41,Yes,0.0008560347,DATETIME,DATETIME,Choice 1-1: 16; Choice 1-2: 6\r
+2,Contest 1,Targeted,41,Yes,<0.0000000001,DATETIME,DATETIME,Choice 1-1: 16; Choice 1-2: 6\r
```

Comment:
i know im always doing this, but do you have an idea of what changed here? i dont think this test case should have hit any of the edge cases this PR is accounting for

Reply:
Ah, I think this may have been a corner case identified above where the weighted error was improperly handled. I'll investigate further.

Reply:
Revisiting this, this contest reports 24 ballots total but has 16 votes for the winner and 10 votes for the loser for a total of 26 votes. That should never happen. 

I dug into it some more, and it appears that the p-value really should be that low. I think the previous result was being affected by the precision bug!

Reply:
shouldn't this have changed with the precision PR then, not this PR?

Reply:
Oh, I see what's happening here. Because we have more ballots in the CVR than the `total_ballots` field, the audit thinks a full-hand recount has happened in the second round, so it sets the p value to 0.

Reply:
So yeah, this test case shouldn't really correspond to real-world, ever. We'd never sample 24 ballots 41 times :joy:

Reply:
ok that makes sense. kind of surprised it didn't just print a p-value of 0 though. but maybe we don't expect that to actually happen so it doesn't matter?

Reply:
Yeah. When we changed the report to just replace any values less than 0.00000001 with that string, we didn't special case p-values of 0. I highly doubt we'll ever be in this situation though.